README.md

```diff
@@ -1,2 +1,136 @@
+platform-specific UI like SwiftUI and Android XML.
+
+The instructions below provide you additional information on how to work with this template and add
+features like navigation, state preservation and more.
+
+The result will be a [Kotlin Multiplatform](https://kotlinlang.org/docs/multiplatform.html) project
+that uses the Decompose framework for common logic with the UI framework(s) of your choice and the
+features you need.
+
+This project was inspired by the [sample project of Decompose](https://github.com/arkivanov/Decompose/tree/master/sample)
+and the [Compose Multipaltform Template](https://github.com/JetBrains/compose-multiplatform-template).
+
+## Getting Started
+
+In order to work with this template you need the following tools:
+* [Android Studio](https://developer.android.com/studio)
+* The [Kotlin Multiplatform Mobile plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile)
+
+Additionally, if you plan to write and run iOS-specific code on simulated or real devices, you need
+the following:
+
+* A machine running a recent version of macOS
+* [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
+
+Tools like [KDoctor](https://github.com/Kotlin/kdoctor) can assist you during the installation
+process.
+
+## Project Structure
+
+The project consists of multiple modules:
+
+### `shared`
+
+This is a Kotlin module that contains the shared logic of all platforms. This also includes the
+Decompose implementation. 
+
+### `compose-ui`
+
+This is a Kotlin module that contains the UI written with Compose Multiplatform that is shared
+across all platforms.
+
+It depends on the `shared` module as it uses the component interfaces from Decompose.
+
+### `app-desktop`
+
+This is a Kotlin module that contains and builds the desktop (JVM) application.
+
+It makes use of the shared code from the modules `shared` and `compose-ui`.
+
+### `app-android`
+
+This is a Kotlin module that contains and builds the Android mobile application.
+
+It makes use of the shared code from the modules `shared` and `compose-ui`.
+
+### `app-ios-compose`
+
+This is a Kotlin module that contains and builds the iOS mobile application.
+
+It makes use of the shared code from the modules `shared` and `compose-ui`.
+
+### `app-ios-swift`
+
+This is a Swift project that build an iOS mobile application.
+
+This module uses only the `shared` module and SwiftUI for its UI (instead of Compose).
+
+> **Note**
+> In practice you normally have either `app-ios-compose` or `app-ios-swift`.
+> 
+> Therefore, do not hesitate to merge them or delete one of the two. 
+
+## Running the project
+
+Depending on the platform you want to build for and run the project on, different gradle tasks may
+be used.
+
+### For Android
+
+When using Android Studio you can simply select `app-android` from the run configurations and run
+the app.
+
+Alternatively you can launch the debug app via
```

Comment:
You are completely right. I was looking an actual gradle task and left that as a placeholder.  realized that it is an adb command that is executed for launching the app.

Should we completely remove the block for Android and expect from the IDE that it will handle it properly, or update it with the ADB command? I think it was something like `adb shell am start ...`

Reply:
Yeah, let's remove this, it's kinda misleading.